### ISSUE TITLE
Detect whether a project uses Maven, exclude otherwise

### ISSUE THREAD

**Issue by Vel1khan** [ashkboos/MavenSecrets#84]
For packages that have repository url in the database, search the content of their repository for `build.gradle` and `build.xml`. This logic should be implemented in the python code before we do the building and after the repository cloning. We can also store two boolean columns on the data base for this:`isGradle` and `isAnt`.

**Comment by mehdikeshani** [comment]
Should we only exclude these from the last step, which is the building part?

**Comment by Vel1khan** [comment]
> Should we only exclude these from the last step, which is the building part?

Yes, or alternatively we can also try to build with Gradle and ANT but it adds additional complexity.

**Comment by mehdikeshani** [comment]
We can use the pom.properties line ending info that we already have to filter non-maven projects.

**Comment by Vel1khan** [comment]
> We can use the pom.properties line ending info that we already have to filter non-maven projects.

I believe this is the way to go. Just checked some Gradle projects and they do not include it. I've looked it up online as well and it seems only Maven should include this. It is of course possible for a plugin to override this default behavior but I don't think it should really be a problem for us. 

There's 2 ways to go about this:
1. Easy way - just exclude all that don't have null in the newline fields (it implies that a pom.properties could not be found, but if we want to analyse other files too later it we will have to adapt and use method 2.
2. Extractor - has_pomproperties. Pretty easy to implement, we just need to run another experiment though.

I would go for 2 tbh

**Comment by mehdikeshani** [comment]
I'm fine with both. Considering your availability, I leave it up to you. The highest priority should be that we have the results as soon as possible, even if we miss some cases we can always rebuild them.
